projects/packages/ip/src/class-ip.php

```diff
@@ -5,7 +5,7 @@
  * @package automattic/jetpack-ip
  */
 
-namespace Automattic\Jetpack;
+namespace Automattic\Jetpack\IP;
```

Comment:
When generating this package with `jetpack generate`, the process defaulted the namespace to `Automattic\Jetpack`. I think using the `Automattic\Jetpack\IP` namespace will be useful considering it seems like most packages use their own namespace to keep class declarations safe. Just dropping this comment to highlight the change for the Crew review :+1:

Reply:
That will mean calling `Automattic\Jetpack\IP\IP` when wanting to use this class; This seems a bit odd to me. 

Would it make more sense for this class to be called `Utils`, for example?

Or if you think this `IP` class will remain one of the only classes in the packages, maybe it can remain under the `Automattic\Jetpack` namespace?